src/main/java/io/anserini/index/IndexReaderUtils.java

```diff
@@ -198,6 +198,15 @@ public static String analyzeTerm(String term, Analyzer analyzer) {
     return termInfo;
   }
 
+  /**
+   * Returns iterator over all terms in the collection.
+   * @param reader index reader
+   * @throws IOException if error encountered during access to index
+   */
+  public static TermsEnum getTermIterator(IndexReader reader) throws IOException {
```

Comment:
I see, you're just exposing the `TermsEnum` directly... does it make sense to wrap in a custom class, like `Term`, which parallels how we wrap the postings?